src/reducers/config.js

```diff
@@ -50,6 +51,9 @@ const config = actionReducer(initialState, {
   SET_USB_FILTER (state, { payload: { usbFilter } }) {
     return state.set('usbFilter', usbFilter)
   },
+  ENABLE_SCHEDULER (state, { payload: { enabled } }) {
```

Comment:
Likewise

Reply:
I'll do the entire file.

Reply:
`ENABLE_SCHEDULER` was removed, but I fixed up the constant use in the file anyway.